themes/theme-next/src/node/config/extendsBundlerOptions.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  addViteConfig,
+  addViteOptimizeDepsExclude,
+  chainWebpack,
+} from '@vuepress/helper'
+import type { App, BundlerOptions } from 'vuepress'
+
+export const extendsBundlerOptions = (
+  bundlerOptions: BundlerOptions,
+  app: App,
+): void => {
+  // ensure theme alias is not optimized by Vite
+  addViteOptimizeDepsExclude(bundlerOptions, app, '@theme')
+  // FIXME: hide sass deprecation warning for mixed-decls
+  const silenceDeprecations = [
+    'mixed-decls',
+    'legacy-js-api',
+    'import',
+    'global-builtin',
```

Comment:
Why are these needed?

Reply:
I forgot to remove the `sass` configuration here; the new theme no longer uses `sass`.